docs/develop/create-sso-office-add-ins-aspnet.md

```diff
@@ -1,14 +1,595 @@
+
+1. In the **Authorized client applications** section, you identify the applications that you want to authorize to your add-in's web application. Each of the following IDs needs to be pre-authorized.
+
+    - `d3590ed6-52b3-4102-aeff-aad2292ab01c` (Microsoft Office)
+    - `ea5a67f6-b6f3-4338-b240-c655ddc3cc8e` (Microsoft Office)
+    - `57fb890c-0dab-4253-a5e0-7188c88b2bb4` (Office on the web)
+    - `bc59ab01-8403-45c6-8796-ac3ef710b3e3` (Office on the web)
+
+    For each ID, take these steps:
+
+    a. Select **Add a client application** button and then, in the panel that opens, set the Client ID to the respective GUID and check the box for `api://localhost:44355/$App ID GUID$/access_as_user`.
+
+    b. Select **Add application**.
+
+1. Select **API permissions** under **Manage** and select **Add a permission**. On the panel that opens, choose **Microsoft Graph** and then choose **Delegated permissions**.
+
+1. Use the **Select permissions** search box to search for the permissions your add-in needs. Select the following. Only the first is really required by your add-in itself; but the `profile` permission is required for the Office host to get a token to your add-in web application. (Only Files.Read.All and profile are actually needed by the add-in. You must request the other two because the MSAL.NET library requires them.)
+
+    * Files.Read.All
+    * offline_access
+    * openid
+    * profile
+
+    > [!NOTE]
+    > The `User.Read` permission may already be listed by default. It is a good practice not to ask for permissions that are not needed, so we recommend that you uncheck the box for this permission if your add-in does not actually need it.
+
+1. Select the check box for each permission as it appears. After selecting the permissions that your add-in needs, select the **Add permissions** button at the bottom of the panel.
+
+1. On the same page, choose the **Grant admin consent for [tenant name]** button, and then select **Yes** for the confirmation that appears.
+
+## Configure the solution
+
+1. In **Visual Studio**, right-click the top node in **Solution Explorer** (the Solution node, not either of the project nodes), and then select **Set startup projects**.
+
+1. Under **Common Properties**, select **Startup Project**, and then **Multiple startup projects**. Ensure that the **Action** for both projects is set to **Start**, and that the project that ends in "...WebAPI" is listed first.
+
+1. Still in **Solution Explorer**, choose the **Office-Add-in-Microsoft-Graph-ASPNETWebAPI** project. In **Properties**, ensure **SSL Enabled** is **True**. Verify that the **SSL URL** is `http://localhost:44355/`.
+
+1. In web.config, use the values that you copied in earlier. Set both the **ida:ClientID** and the **ida:Audience** to your **Application (client) ID**, and set **ida:Password** to your client secret.
```

Comment:
Nit: For me, the file was capitalized "Web.Config". I think it should also be in quotes to be consistent with your other file references.